src/module.c

```diff
@@ -1795,10 +1795,13 @@ int RM_HashSet(RedisModuleKey *key, int flags, ...) {
         if (flags & REDISMODULE_HASH_CFIELDS)
             low_flags |= HASH_SET_TAKE_FIELD;
         updated += hashTypeSet(key->value, field->ptr, value->ptr, low_flags);
-        field->ptr = NULL; /* Ownership is now of hashTypeSet() */
-
-        /* Cleanup */
-        if (flags & REDISMODULE_HASH_CFIELDS) decrRefCount(field);
+        
+        /* If CFIELDS is active, ownership is now of hashTypeSet() */
+        if (flags & REDISMODULE_HASH_CFIELDS) {
+           field->ptr = NULL; 
+           /* Cleanup */
+           decrRefCount(field);
+        }
```

Comment:
that's how @antirez usually writes - one liners braceless, multiline in braces.  and it's usually mixed in the same function. for example: 

https://github.com/dvirsky/redis/blob/cfaef8d5d494dc199117cb7a6841dfbdc43dd22e/src/module.c#L1198